bats/core/api/onchain-send.bats

```diff
@@ -4,6 +4,7 @@ load "../../helpers/cli.bash"
 load "../../helpers/user.bash"
 load "../../helpers/onchain.bash"
 load "../../helpers/wallet.bash"
+load "../../helpers/dealer.bash"
```

Comment:
dealer is not the right name for the file where `balance_for_check` comes from.
Perhaps ledger?